### DISCUSSION TITLE
Docker error

### DISCUSSION THREAD

**Discussion by halogen28372** [blakeblackshear/frigate#3187] [General]
Hello all,

I am trying to install Frigate on an Atomic Pi running Ubuntu server.  I have created a docker-compose like this:

version: "3.9"
services:
  frigate:
    container_name: frigate_new
    privileged: true # this may not be necessary for all setups
    restart: unless-stopped
    image: blakeblackshear/frigate:stable-amd64
    shm_size: "64mb" # update for your cameras based on calculation above
    devices:
      - /dev/dri/renderD128 # for intel hwaccel, needs to be updated for your hardware
    volumes:
      - /etc/localtime:ro
      - /frigate/config.yml:ro
      - /frigate/media
      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    ports:
      - "5000:5000"
      - "1935:1935" # RTMP feeds
    environment:
      FRIGATE_RTSP_PASSWORD: "TOP_Seceret"

The problem is when I run "docker-compose up" it gives me the following error:

ERROR: Duplicate mount points: [/etc/localtime:ro:rw, /frigate/config.yml:ro:rw]

Does anyone know what may be causing this?

**Comment by trythatagain** [comment]
Where did you get this docker-compose file from? This is never going to work, have a look at the Frigate docs and perhaps a few docker videos. You've not got any mapping in your volumes, in the docs it's   :-
volumes:
      - /path/to/your/config.yml:/config/config.yml:ro
      - /path/to/network/storage:/media/frigate
Where the right side is your PC's settings and teh right side after the ':' is the docker container paths.

If you search the discussions you will find a number of docker-compose files that should get you started. Again the Frigate docs are a great place to start.
